resources/tickets.py

```diff
@@ -86,9 +87,24 @@ def get(self):
             return {"tickets": [ticket.json() for ticket in TicketModel.query.all()]}
 
     @pm_level_required
-    def post(self):
+    def put(self):
         data = Tickets.parser.parse_args()
         ticket = TicketModel(**data)
 
         ticket.save_to_db()
         return ticket.json(), 201
+
+    @pm_level_required
+    def post(self):
+        data = Tickets.parser.parse_args()
+
+        if not ("ids" in data and type(data["ids"]) is list):
+            return {"message": "Ticket IDs missing in request"}, 400
+
+        for id in data["ids"]:
+            ticket = TicketModel.find_by_id(id)
+            if not ticket:
+                return {"message": "Ticket cannot be deleted"}, 400
+            ticket.delete_from_db()
```

Comment:
This looks like it should be a delete action instead of a post action. Can you explain these changes